### PR TITLE
Add nginx exporter

### PR DESCRIPTION
**Pull request by maximxmoroz**
Add nginx-exporter and make major changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-      - ./volumes/certbot/www:/var/www
-      - ./volumes/certbot/letsencrypt:/etc/letsencrypt
   bostrom:
     image: cyberd/bostrom:dragonberry-cuda11.4
     restart: always
     container_name: bostrom
     ports:
-      - "26090:9090"
-      - "26656:26656"
+      - "26091:9090"
```
Comment (mrlp4) [inline docker-compose.yml:9]: Why do we have 2 different external ports mapped to same 9090 internal port?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
+      - "36656:26656"
```
Comment (mrlp4) [inline docker-compose.yml:10]: Same here: 26656 & 36656 external ports mapped to same 26656 internal port.

File: volumes/nginx/cybernode.conf [hunk 3]
```diff
@@ -14,15 +14,25 @@ server {
...
         listen 443 ssl;
 
-        root /var/www/html/;
+        root /var/www/;
 
         ssl_certificate_key /etc/letsencrypt/live/cybernode/privkey.pem;
         ssl_certificate /etc/letsencrypt/live/cybernode/fullchain.pem;
 
-        server_name rpc.${DOMAIN};
+        server_name 1rpc.${DOMAIN};
```
Comment (mrlp4) [inline volumes/nginx/cybernode.conf]: Why endpoint named `1rpc` ?

**Comment by maximxmoroz** [comment]
Сhange the name of the rpc endpoint

**Review by mrlp4 — APPROVED** [review]

**Pull request merged**